docker-compose.yml

```diff
@@ -9,7 +9,9 @@ services:
     volumes:
       - ./app:/app
     command: >
-      sh -c "python manage.py runserver 0.0.0.0:8000"
+      sh -c "python manage.py wait_for_db && 
```

Comment:
this can we solved with https://github.com/vishnubob/wait-for-it as well or even https://github.com/palfrey/wait-for-db